src/components/shared/ButtonWithPanel.js

```diff
@@ -85,14 +89,23 @@ class ButtonWithPanel extends React.PureComponent<Props, State> {
       this._panel.open();
     }
   }
+
   closePanel() {
     if (this._panel && this.state.open) {
       this._panel.close();
     }
   }
 
-  _onButtonClick = () => {
-    this.openPanel();
+  _onWindowClick = () => {
+    this.closePanel();
+  };
+
+  _onButtonClick = (e: SyntheticMouseEvent<>) => {
```

Comment:
The error in the tests tells you that this variable is unused. (previously we were using it to call `stopPropagation` but we're not doing it anymore). So you can remove it.

Reply:
I had previously also tried that @julienw and still it is giving the same error. It's some `_defaultview` issue. Not sure what it is.